src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -1,10 +1,22 @@
 import { ChatHeaderStyled } from './ChatHeaderStyled';
 
-const ChatHeader = ({ countryName, participantsAmount }) => (
+const ChatHeader = ({
+  chatName,
+  participantsAmount,
+  selectedCompanion,
+  isPrivateChat,
+}) => (
   <ChatHeaderStyled>
-    <h5>{countryName || 'Country Name'}</h5>
-    <p>{`${participantsAmount || '0'} participants`}</p>
+    {isPrivateChat ? (
+      <h5>{selectedCompanion.userName}</h5>
+    ) : (
+      <h5>{chatName || 'Country Name'}</h5>
+    )}
```

Comment:
It could be one h5 tag with conditionaly rendered text... (Dima left me the same comment so now I imagine that I have a lot of expirience and left same to you :) )

Reply:
Done